GPU/GPUTracking/Merger/GPUTPCGMTrackParam.cxx

```diff
@@ -308,6 +308,20 @@ GPUd() bool GPUTPCGMTrackParam::Fit(GPUTPCGMMerger* GPUrestrict() merger, int iT
         }
 #endif
         retVal = prop.Update(yy, zz, clusters[ihit].row, param, clusterState, rejectChi2, &interpolation.hit[ihit], refit);
+
+        if (o2::utils::DebugStreamer::checkStream(o2::utils::StreamFlags::streamUpdateTrack)) {
+          mStreamer.setStreamer("debug_update_track", "UPDATE");
+          mStreamer.getStreamer() << mStreamer.getUniqueTreeName("tree_update_track").data()
```

Comment:
Can we remove the newlines and put alll the `<<` in a single line, to reduce the number of lines used by the debug code?